internal/bminventory/inventory.go

```diff
@@ -4237,6 +4237,9 @@ func (b *bareMetalInventory) uploadLogs(ctx context.Context, params installer.Up
 		if err != nil {
 			return err
 		}
+		b.eventsHandler.AddEvent(ctx, params.ClusterID, params.HostID, models.EventSeverityInfo,
+			fmt.Sprintf("uploaded logs for host %s cluster %s to bucket %s",
```

Comment:
I don't think that we need to use the host ID in the message because it's will already be in the event structure, maybe you should use `hostutil.GetHostnameForMsg(h)` to get hostname?

Reply:
Done